<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-03-20T23:40:49 EDT" name="Suite" finished-at="2025-03-20T23:41:12 EDT" duration-ms="23071">
    <groups>
    </groups>
    <test started-at="2025-03-20T23:40:49 EDT" name="Test" finished-at="2025-03-20T23:41:12 EDT" duration-ms="23071">
      <class name="MainTests.OptimizedE2E">
        <test-method is-config="true" signature="getPage()[pri:0, instance:MainTests.OptimizedE2E@23e84203]" started-at="2025-03-20T23:40:49 EDT" name="getPage" finished-at="2025-03-20T23:41:00 EDT" duration-ms="11066" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getPage -->
        <test-method signature="OE2E()[pri:0, instance:MainTests.OptimizedE2E@23e84203]" started-at="2025-03-20T23:41:00 EDT" name="OE2E" finished-at="2025-03-20T23:41:10 EDT" duration-ms="10078" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at MainTests.OptimizedE2E.OE2E(OptimizedE2E.java:26)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OE2E -->
        <test-method is-config="true" signature="addResultsToTestRail(org.testng.ITestResult)[pri:0, instance:MainTests.OptimizedE2E@23e84203]" started-at="2025-03-20T23:41:11 EDT" name="addResultsToTestRail" finished-at="2025-03-20T23:41:12 EDT" duration-ms="1419" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=OE2E status=FAILURE method=OptimizedE2E.OE2E()[pri:0, instance:MainTests.OptimizedE2E@23e84203] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addResultsToTestRail -->
      </class> <!-- MainTests.OptimizedE2E -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
